Shopify/dawn
# [Slideshow] Add ambient movement to background

### PR Summary: 

Adds an option for ambient animation to the Slideshow section's background images.

https://user-images.githubusercontent.com/1202812/224727458-4fa51b8f-11d1-4d9a-8f39-81dccf4bcfa6.mp4

### Why are these changes introduced?

Closes #2341. 

### What approach did you take?

Uses the same approach (and existing CSS) that was added in https://github.com/Shopify/dawn/pull/2342.

This implements a new dropdown menu [as shown in the ticket](https://github.com/Shopify/dawn/pull/2341). When "Ambient movement" is selected, a CSS class is applied to the media elements within the Image Banner, triggering a long-duration CSS animation that slowly moves the image around within its frame. As this feature is purely aesthetic, it supports the visitor's prefers-reduced-motion preference.

Like https://github.com/Shopify/dawn/pull/2342, it also conditionally loads in a slightly higher-res image when the ambient animation is active, in order to account for the scale effect that's applied to the image. 

### Visual impact on existing themes

This feature is off by default. When it's turned on, all background images in the slideshow will animate ambiently. 

### Testing steps/scenarios

1. [Visit the demo store](https://os2-demo.myshopify.com/admin/themes/139517362198/editor).
2. Add and select a Slideshow.
3. Look for the new "Image behavior" dropdown underneath the new "Animations" header in the Slideshow's sidebar.
4. Select "Ambient movement" from that dropdown.
5. Add image backgrounds to individual slides, verify that they appear as expected.
6. Try various breakpoints, verify that they appear as expected.

### Checklist
